### PR TITLE
[iOS] Add missing test for Vertical Stretch

### PR DESCRIPTION
**Pull request by almedina-ms**
Adds a missing file that was included in the solution but wasn't added to the commit. This missing file may make the build process to fail

### CODE REVIEW

**Comment by nesalang** [comment]
adding this won't affect UWP's test app outputs, right?

File: samples/Tests/InputsFormWithHeightStretch.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+				},
+				{
+					"type": "Column",
+					"width": 1,
+					"items": [
+						{
+							"type": "Image",
+							"url": "https://upload.wikimedia.org/wikipedia/commons/b/b2/Diver_Silhouette%2C_Great_Barrier_Reef.jpg",
+							"size": "auto",
+                            "height": "stretch"
```
Comment (nesalang) [inline samples/Tests/InputsFormWithHeightStretch.json]: tabs?

**Comment by paulcam206** [comment]
Is this what's causing the build break on #1611?

**Comment by nesalang** [comment]
i think so

---
In reply to: [397431440](https://github.com/Microsoft/AdaptiveCards/pull/1603#issuecomment-397431440) [](ancestors = 397431440)

**Comment by almedina-ms** [comment]
@paulcam206 #1611 is unrelated, the issue was that the PR that I passed yesterday included a file in the solution that I forgot to add to the commit, so now if you try to build the iOS solution then you get the file missing error

**Pull request merged**